hazelcast/src/main/java/com/hazelcast/instance/BuildInfoProvider.java

```diff
@@ -58,7 +58,7 @@ public static BuildInfo getBuildInfo() {
         } else {
             build = String.valueOf(hazelcastBuild);
         }
-        int buildNumber = Integer.parseInt(build);
+        int buildNumber = 0; //Integer.parseInt(build);
```

Comment:
sorry temporarily changed it.  shouldn't have included in the commit.  will restore.